### PR TITLE
LRQA-82554

### PR DESCRIPTION
**Pull request by Claudioss23**

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by Claudioss23** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_HEIGHT</td>
+	<td>//div[contains(@id,'customSizeSelector')]//input[contains(@id,'height')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_WIDTH</td>
+	<td>//div[contains(@id,'customSizeSelector')]//input[contains(@id,'width')]</td>
+	<td></td>
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path:31]: Hey, here you can see that the only difference between the xpaths is the id of the input, correct? 
What you did is right, but if you want to not repeat yourself what you can do is create only one xpath for these two cases and put a variable that can assume one of the values you are looking for: the height or the width. Like this:

`//div[contains(@id,'customSizeSelector')]//input[contains(@id,'${nameOfTheVariable}')]`

Tell me if this works, thank you!

btw If you choose to create only one xpath, you will need to change the name of the XPath, you can put CUSTOM_SIZE_INPUT or any other name of your choice.

File: portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_APPLY_BUTTON</td>
+	<td>//div[contains(@id,'customSizeSelector')]//button[@aria-label='Apply Custom Size']</td>
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path:35]: Hey, I think you could also try using the XPath like this:
`//button[@aria-label='Apply Custom Size']`
What do you think? 
Thanks!

File: portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyPath</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>SIMULATION_MENU</td>
+	<td>//div[contains(@class,'control-menu')]//button[@data-qa-id='simulation']</td>
+	<td></td>
```
[the remark this answers is not in the document]
Comment (Claudioss23) [reply portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path:16]: Alright. Thanks for reviewing!

File: portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_BUTTON</td>
+	<td>//div[contains(@class,'row size-selector')]//button[child::span[text()='Custom']]</td>
```
[the remark this answers is not in the document]
Comment (Claudioss23) [reply portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path:20]: That worked. Thanks for the review!

File: portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path (hunk 1 above)
Comment (Claudioss23) [reply portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path:31]: That's a good idea! Thanks for the advice. I tested it and it worked nicely. Thank you!

File: portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path (hunk 2 above)
Comment (Claudioss23) [reply portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path:35]: Okay, I changed it. Thanks!

**Comment by Claudioss23** [comment]
ci:test:sf

**Review by manuferreira — APPROVED** [review]
Hey, thanks for working on it and for the hard work! :muscle:

**Pull request closed without merging**